libs/common-const/src/networks.ts

```diff
@@ -1,32 +1,41 @@
 import { SupportedChainId } from '@cowprotocol/cow-sdk'
 import { JsonRpcProvider } from '@ethersproject/providers'
 
-function initRpcUrls(): Record<SupportedChainId, string> {
-  const REACT_APP_INFURA_KEY = process.env.REACT_APP_INFURA_KEY
-  const REACT_APP_NETWORK_URL_1 = process.env.REACT_APP_NETWORK_URL_1
-  const REACT_APP_NETWORK_URL_5 = process.env.REACT_APP_NETWORK_URL_5
-  const REACT_APP_NETWORK_URL_100 = process.env.REACT_APP_NETWORK_URL_100
-
-  if (!REACT_APP_INFURA_KEY && !(REACT_APP_NETWORK_URL_1 && REACT_APP_NETWORK_URL_5 && REACT_APP_NETWORK_URL_100)) {
-    throw new Error(
-      `Either REACT_APP_INFURA_KEY or REACT_APP_NETWORK_URL_{1,5,100} environment variables must be defined`
-    )
-  }
-
-  return {
-    [SupportedChainId.MAINNET]: REACT_APP_NETWORK_URL_1 || `https://mainnet.infura.io/v3/${INFURA_KEY}`,
-    [SupportedChainId.GNOSIS_CHAIN]: REACT_APP_NETWORK_URL_100 || `https://rpc.gnosis.gateway.fm`,
-    [SupportedChainId.GOERLI]: REACT_APP_NETWORK_URL_5 || `https://goerli.infura.io/v3/${INFURA_KEY}`,
-  }
-}
-
 const INFURA_KEY = process.env.REACT_APP_INFURA_KEY
-if (typeof INFURA_KEY === 'undefined') {
-  throw new Error(`REACT_APP_INFURA_KEY must be a defined environment variable`)
+
+const DEFAULT_RPC_URL: Record<SupportedChainId, { url: string; usesInfura: boolean }> = {
+  [SupportedChainId.MAINNET]: { url: `https://mainnet.infura.io/v3/${INFURA_KEY}`, usesInfura: true },
+  [SupportedChainId.GNOSIS_CHAIN]: { url: `https://rpc.gnosis.gateway.fm`, usesInfura: false },
+  [SupportedChainId.GOERLI]: { url: `https://goerli.infura.io/v3/${INFURA_KEY}`, usesInfura: true },
```

Comment:
Shouldn't it be prefixed with react_app like before? `REACT_APP_INFURA_KEY`

Reply:
No, because it's the name of the const, not the env:
`const INFURA_KEY = process.env.REACT_APP_INFURA_KEY`